openedx/frontend-app-authoring
# feat: update ora settings to only be flexible peer grading

## Description

This change impacts the Course Author. This PR improves the clarity of the ORA Settings card in the pages and resources page. The only ORA setting available for configuration is peer grading assessments. This changes brings this setting to the top level of the card because there are no current plans to add other ORA configurations. Moving the setting to the top level makes it clear to the user which settings are available.

<table>
<tr>
<th>AC</th>
<th>Before</th>
<th>After</th>
</tr>
<tr>
<td>1</td>
<td><img width="796" alt="Screenshot 2024-09-26 at 9 58 02 AM" src="https://github.com/user-attachments/assets/06a11222-a86d-43a4-a52e-ac95e7622815"></td>
<td>
<img width="280" alt="Screenshot 2024-09-26 at 9 59 21 AM" src="https://github.com/user-attachments/assets/8801ed6b-2873-411f-9aa6-fea6ff1a407a"></td>

</tr>
<tr>
<td>2</td>
<td>
<img width="733" alt="Screenshot 2024-09-26 at 10 01 00 AM" src="https://github.com/user-attachments/assets/4353835c-ca59-4a1e-b20d-3c1a4e57ee13">
</td>
<td>
<img width="733" alt="Screenshot 2024-09-26 at 10 01 00 AM" src="https://github.com/user-attachments/assets/d67260b3-44b2-49ea-ba62-1a26bd8f7ce3"></td>
</tr>
<tr>
<td>3</td>
<td>
<img width="798" alt="Screenshot 2024-09-26 at 10 26 33 AM" src="https://github.com/user-attachments/assets/57d6a622-e18d-47fb-8a8b-f3be74776b81"></td>
<td>
<img width="796" alt="Screenshot 2024-09-26 at 9 58 02 AM" src="https://github.com/user-attachments/assets/f668ea64-e766-4f70-96d9-d92796ab96db"></td>
</tr>

</table>

## Supporting information

JIRA Ticket: [AU-2058 🔒 ](https://2u-internal.atlassian.net/browse/AU-2058)

> Update ORA card in pages and resources for clarity
>
> AC 1 
> The front of the card should read:
> title: Flexible Peer Grading for ORAs
> blurb: Course level settings for Flexible Peer Grading in Open Response Assessments 
> 
> AC 2
> “inside” of card should read:
> title: Configure Flexible Peer Grading for ORAs
> 
> AC 3
> “enabled” tag should hide/show based on true value of toggle inside card 

## Testing instructions

1. Open a course that allow ORA blocks
2. Navigate to the Pages and Resources page
3. Confirm that the former ORA card now say "Flexible Peer Grading"
4. In another tab open the advanced settings for the course
5. Update "Force on flexible peer open assessments" to `true`
6. Reload the Pages and Resources page and confirm that the "Flexible Peer Grading" has the "enabled" badge
7. Click the Settings icon (gear)
8. Confirm that the modal loads and the toggle is enabled.
9. Turn off the toogle
10. Click "Save"
11. The modal should close
12. Confirm that the "Enabled" badge no longer appears on the card.

## Other information

This PR is dependent on edx-plaform [#35550](https://github.com/openedx/edx-platform/pull/35550)
